sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -618,5 +618,16 @@ object DateTimeUtils {
       date + daysInNormalYear(month - 1) - dayOfMonth
     }
   }
-
+  /**
+   * Returns the truncate level, could be [[Calendar.MONTH]]/[[Calendar.YEAR]]/-1
```

Comment:
It's not clear that Calendar.MONTH is different than -1, why not just define them here?